android.html

```diff
@@ -22,7 +22,7 @@
   <body>
     <header class="navbar-primary navbar">
       <section class="navbar-section" >
-        <a href="https://banglejs.com" target="_blank" class="navbar-brand mr-2" ><img src="img/banglejs-logo-sml.png" alt="Bangle.js">
+        <a href="https://banglejs.com" target="_blank" class="navbar-brand mr-2"><img src="img/banglejs-logo-small.svg" alt="Bangle.js">
```

Comment:
Do we want to delete `banglejs-logo-sml.png`?

Reply:
Maybe don't bother? It's not like it's a big file...

Reply:
I feel like if it were to be deleted it would be more a matter of semantics

Reply:
Yeah my thought was that someone might change one without the other